src/gw_cpu_kernel.cpp

```diff
@@ -29,7 +29,7 @@
 namespace green::mbpt::kernels {
 
   void gw_cpu_kernel::solve(G_type& g, St_type& sigma_tau) {
-    _coul_int1 = new df_integral_t(_path, _nao, _NQ, _bz_utils);
+    _coul_int1 = new df_integral_t(_path, _nao, _NQ, _bz_utils, _verbose);
```

Comment:
will be very verbose on all nodes. Propose only passing in _verbose on node zero.